ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerApiServer.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.infrastructure
+
+import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantTestContext
+import com.digitalasset.ledger.api.v1.command_completion_service.CommandCompletionServiceGrpc.CommandCompletionService
+import com.digitalasset.ledger.api.v1.transaction_service.TransactionServiceGrpc.TransactionService
+
+object LedgerApiServer {
+
+  /** temporary work-around to get access to the transaction and command completion service
+    * while we haven't merged the test into the upstream code
+    */
```

Comment:
I have a feeling you don't need this in this code base.

Reply:
It looks like we do or I can add some more methods to the participant test context.

Reply:
Exactly, do that! 😀